src/transformers/utils/generic.py

```diff
@@ -867,3 +869,48 @@ class LossKwargs(TypedDict, total=False):
     """
 
     num_items_in_batch: Optional[int]
+
+
+def is_timm_hub_checkpoint(pretrained_model_name_or_path: str) -> bool:
```

Comment:
Given the method's objective, I would have it accept only a `pretrained_model_name`

Reply:
function removed with https://github.com/huggingface/transformers/commit/ff6efde8d0d77baa83e9e7a264aec84618751d1b (see comment below)